sensing/autoware_pointcloud_preprocessor/src/pointcloud_accumulator/pointcloud_accumulator_node.cpp

```diff
@@ -23,9 +23,8 @@ PointcloudAccumulatorComponent::PointcloudAccumulatorComponent(const rclcpp::Nod
 {
   // set initial parameters
   {
-    accumulation_time_sec_ = static_cast<double>(declare_parameter("accumulation_time_sec", 2.0));
-    pointcloud_buffer_.set_capacity(
-      static_cast<size_t>(declare_parameter("pointcloud_buffer_size", 50)));
+    accumulation_time_sec_ = declare_parameter<double>("accumulation_time_sec");
+    pointcloud_buffer_.set_capacity(declare_parameter<int64_t>("pointcloud_buffer_size"));
```

Comment:
The boost class uses size_t it seems
https://www.boost.org/doc/libs/1_42_0/libs/circular_buffer/doc/circular_buffer.html

Reply:
@knzo25 thanks! However, declare_parameter<size_t> doesn't work for ros2 parameter. (compile will fatiled)
![image](https://github.com/user-attachments/assets/b53d23ea-454b-4bc3-858a-4f7d1c28ec04)

reference: https://autowarefoundation.github.io/autoware-documentation/main/contributing/coding-guidelines/ros-nodes/parameters/

Reply:
*compile fails

I know that size_t is incompatible (internally it struggles with signedness). As of this PR there is an implicit cast which needs to made explicit. This applies to all other PRs

Reply:
Thanks, fixed in https://github.com/autowarefoundation/autoware.universe/pull/8487/commits/e125de73b329a76be7c625ce1cf11a3fb01f2f16